xp0-double/xp0d-azuredeploy-msdeploy.json

```diff
@@ -190,9 +190,7 @@
                   "Application Insights Instrumentation Key": "[reference(concat('Microsoft.Resources/deployments/', 'sitecore-infra'), '2015-11-01').outputs.appInsightsInstrumentationKey.value]",
                   "Application Insights Role": "CM",
                   "KeepAlive Url": "[concat('https://', reference(resourceId('Microsoft.Web/sites', variables('cmWebAppNameTidy'))).hostNames[0], '/sitecore/service/keepalive.aspx')]",
-                  "License Xml": "[variables('licenseXml')]",
-                  "IP Security Client IP": "[parameters('security_clientIp')]",
```

Comment:
My set isn't U3 yet, did not have time to check U2 vs U3 changes. But I would actually like to use these params, and if you leave the params out of your param file, they will be ignored. So please do not remove them.